viewer/src/components/PlayerBoard/BuildingGroup.vue

```diff
@@ -117,7 +119,16 @@ export default class BuildingGroup extends Vue {
   ac2: boolean;
 
   get board() {
-    return this.player.board ?? factionBoard(this.faction, this.player.factionVariant);
+    if (this.player.board) {
+      return this.player.board;
+    }
+    const engine = this.$store.state.gaiaViewer.data as Engine;
+    const customization: FactionCustomization = {
+      players: engine.players.length,
+      variant: engine.options.factionVariant ?? "standard",
+    };
+    const factionVariant = factionVariantBoard(customization, this.faction);
```

Comment:
```suggestion
    const factionVariant = factionVariantBoard(engine.customization, this.faction);
```

The engine in the store is actually instantiated iirc so getters work